### PR TITLE
add OAM to kpt ecosystem

### PR DESCRIPTION
**Pull request by wonderflow**
Demonstrate how to use kpt manage applications defined by [oam](https://github.com/oam-dev/spec) and [Crossplane](https://github.com/crossplane/crossplane)

### CODE REVIEW

**Comment by wonderflow** [comment]
@googlebot I signed it!

**Comment by pwittrock** [comment]
Thanks so much!  I am out the rest of this week, but will take a look Monday.

Check out the docs contributing instructions to see how to generate the site and view it locally:

https://github.com/GoogleContainerTools/kpt/blob/master/CONTRIBUTING.md#contributing-to-docs

File: site/content/en/guides/ecosystem/oam/_index.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+component.core.oam.dev/test-component created
+3 resource(s) applied. 2 created, 0 unchanged, 1 configured
+configmap/grouping-object-a4e18a93 is Current: Resource is always ready
+applicationconfiguration.core.oam.dev/example-appconfig is Current: Resource is current
+resources failed to the reached Current status
+configmap/grouping-object-a4e18a93 pruned
+2 resource(s) pruned
+```
+
+Currently `kpt live apply` seems not work well and will be fixed soon by [issue #498](https://github.com/GoogleContainerTools/kpt/issues/498).
```
Comment (pwittrock) [inline site/content/en/guides/ecosystem/oam/_index.md:209]: The issue seems to be resolved.  Can you check if has been fixed in the latest release, then we could remove this line?

Comment (wonderflow) [reply site/content/en/guides/ecosystem/oam/_index.md]: yeah, I tried and removed that line. Thanks!

**Comment by wonderflow** [comment]
@pwittrock  Do I need to run `make docs` and commit in this PR? I found it's a big PR if I doing so.

**Comment by pwittrock** [comment]
@wonderflow 

Submit without make docs to avoid the conflicts, and I will run make docs in a follow up PR.  WDYT?

**Comment by pwittrock** [comment]
@wonderflow 

Submit without make docs to avoid the conflicts, and I will run make docs in a follow up PR.  WDYT?

**Comment by wonderflow** [comment]
@pwittrock  sure, let me fix it right now

**Comment by pwittrock** [comment]
Thanks!

**Pull request merged**